pmd-apex/src/test/java/net/sourceforge/pmd/lang/apex/ast/ASTUserInterfaceTest.java

```diff
@@ -28,4 +28,12 @@ public void testInnerInterfaceName() {
         Assert.assertNotNull(innerNode);
         Assert.assertEquals("Bar", innerNode.getImage());
     }
+
+    @Test
+    public void testSuperInterface() {
+        ApexNode<?> node = parse("public interface CustomInterface extends A, B implements C {}");
```

Comment:
@rsoesemann is this even valid Apex? I'm surprised it's being parsed by Jorje…

Reply:
hm... looks strange indeed. Not sure, what I thought back then.... the interface should not implement C, right?

Reply:
About jorje not complaining - we ignore all errors from jorje - see #1741

Reply:
I've updated the example.